src/librustc/ty/layout.rs

```diff
@@ -943,7 +943,7 @@ impl<'a, 'tcx> LayoutDetails {
             /// A univariant, the last field of which may be coerced to unsized.
             MaybeUnsized,
             /// A univariant, but part of an enum.
```

Comment:
Can you change the comment to say "with a prefix of an arbitrary size & alignment (e.g. enum tag)" instead of "but part of an enum"?